src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -9239,10 +9239,10 @@ namespace WaterThermalTanks {
 			if ( WaterThermalTank( WaterThermalTankNum ).FirstRecoveryDone && WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel > 0.0 ) {
 				// Calculate Recovery Efficiency based on energy used to recover from the first draw
 				// FirstRecoveryFuel is recorded inside the CalcWaterThermalTank subroutine
-				RecoveryEfficiency = DrawMass * CPHW( 57.2222 ) * ( 57.2222 - 14.4444 ) / WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel;
+				RecoveryEfficiency = DrawMass * CPHW() * ( 57.2222 - 14.4444 ) / WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel;
 
 				// Calculate Energy Factor based on total energy (including parasitics) used over entire test
-				EnergyFactor = TotalDrawMass * CPHW( 57.2222 ) * ( 57.2222 - 14.4444 ) / FuelEnergy;
+				EnergyFactor = TotalDrawMass * CPHW() * ( 57.2222 - 14.4444 ) / FuelEnergy;
```

Comment:
If Cp is ever corrected to use actual values, it won't be so easy to remember what values should be passed. Do other models use a constant? If most Cp calls were converted to fluid properties, should this be done everywhere? And then we will have to think about what values to pass since we have now removed all arguments.